melodic/distribution.yaml

```diff
@@ -2704,6 +2704,20 @@ repositories:
       url: https://github.com/ros/ros_environment.git
       version: melodic
     status: maintained
+  ros_fmu:
+    release:
+      packages:
+      - ros_fmu
```

Comment:
What do you think about `fmi_adapter`? FMI is the official acronym of the Functional Mock-Up Interface standard and is hit no. 2 in a Google search. _Adapter_ precisely describes the main functionality provided by this package. `functional_mock-up_interface_adapter` would be a bit lengthy. If you agree on `fmi_adapter`, I would contact the FMI standardization organization as FMI is actually protected by a trademark, cf. [https://svn.fmi-standard.org/fmi/branches/public/docs/Trademark%20Guidelines%20for%20the%20Use%20of%20the%20FMI%20Trademark%20V1.0.pdf](https://svn.fmi-standard.org/fmi/branches/public/docs/Trademark%20Guidelines%20for%20the%20Use%20of%20the%20FMI%20Trademark%20V1.0.pdf).

Reply:
`fmi_adapter` does seem to quite accurately describe what it's doing so that would be a good name.

From that document:
```
The use of the letters FMI in a product name or other relevant reference is allowed when:
The use is clearly a reference to adhering to the standard
```
As long as you are planning ti implement the standard as defined above and not do the prohibited uses I don't think there's a problem referring to it by name.

Reply:
I have the same opinion regarding the use of FMI. To be on the safe side, I'll drop them a mail and then fix this pull request.

Reply:
Got a positive reply from the FMI project leader regarding the name. I'll close this pull request and create a new one using bloom for fmi_adapter.